src/update_xs_yum.ml

```diff
@@ -470,27 +469,26 @@ let get_last_successful_build branch =
     (get_env_var "JENKINS_URL") // path
   in
   get_http_body url
-  >>|= fun body ->
+  >>= fun body ->
   let open Yojson.Basic in
   Lwt.catch
     (fun () ->
-      `Ok (from_string body
-          |> Util.member "lastSuccessfulBuild"
-          |> Util.member "number"
-          |> to_string)
-      |> Lwt.return)
+       (from_string body
+        |> Util.member "lastSuccessfulBuild"
+        |> Util.member "number"
+        |> to_string)
+       |> Lwt.return)
```

Comment:
If you have the parentheses why not `Lwt.return (...)`? Otherwise why can't you get rid of the parentheses and do 
```ocaml
from_string body
|> Util.member "lastSuccessfulBuild"
|> Util.member "number"
|> to_string
|> Lwt.return
```

Reply:
Makes sense, thanks for the suggestion.

Reply:
Done.